lib/hanami/hanamirc/parser.rb

```diff
@@ -0,0 +1,63 @@
+module Hanami
+  class Hanamirc
+    module Parser
```

Comment:
@aderyabin I don't think this is necessary. This is the `Dotenv::Parser`, which needs to handle edge cases because of human editing of `.env` files.

Our `.hanamirc` is stricter and I think that the previous, simpler implementaion (`split("=")`) was enough. This because our file isn't meant to be edited **and** it doesn't have complex values that include double quotes or UNIX `export` keyword.